src/Card/Card.js

```diff
@@ -16,7 +16,7 @@ const StyledCard = createComponent({
   `,
 });
 
-const Card = props => <StyledCard {...props} />;
+const Card = React.forwardRef((props, ref) => <StyledCard ref={ref} {...props} />);
```

Comment:
`styled-components` will forward refs for you but if you want the ref to be on `Box` instead of the wrapped `StyledCard` you might have to verify that `createComponent` does the right thing. could be worth adding a test in that case

Reply:
Thanks, I will add a test to ensure that the ref is assigned as expected.

Reply:
This is right - functional components don't forward refs out of the box and this ref should be passed directly to Box. We're wrapping some components as a [hack](https://github.com/pedronauck/docz/issues/156#issuecomment-411002811) for the <PropsTable /> component to work for styled-components. All documentation generators (storybook, docz, stylguidist) are affected due to the issue in the underlying react-docgen parser. Searching for better solutions